src/mongo_client.ts

```diff
@@ -528,7 +528,7 @@ export class MongoClient extends TypedEventEmitter<MongoClientEvents> {
   startSession(options?: ClientSessionOptions): ClientSession {
     options = Object.assign({ explicit: true }, options);
     if (!this.topology) {
-      throw new MongoDriverError('Must connect to a server before calling this method');
+      throw new MongoNotConnectedError('Must connect to a server before calling this method');
```

Comment:
Let's take the chance to remove server from the message.
`MongoClient must be connected to start a session`